classes/handlers/keywordsBehaviorHandler.ts

```
@@ -326,14 +326,15 @@ const scriptHandlers = {
           }
 
           if (historyLoaded && !hadRawHistory && text) {
-            generateSessionTitle(prompt, text)
-              .then((title) => {
+            (message.client as any).db.aiChat.getHistory(aiSession.sessionId, 100)
+              .then((history: { role: string; message: string }[]) => generateTitleForHistory(history))
```

ESLint

[error] 'history' is already declared in the upper scope on line 139 column 9.

(no-shadow)